pkg/reconciler/accessor/istio/virtualservice.go

```diff
@@ -64,7 +70,7 @@ func ReconcileVirtualService(ctx context.Context, owner kmeta.Accessor, desired
 		return nil, kaccessor.NewAccessorError(
 			fmt.Errorf("owner: %s with Type %T does not own VirtualService: %q", owner.GetName(), owner, name),
 			kaccessor.NotOwnResource)
-	} else if !equality.Semantic.DeepEqual(vs.Spec, desired.Spec) {
+	} else if hasDesiredDiff(vs, desired) {
 		// Don't modify the informers copy
 		existing := vs.DeepCopy()
 		existing.Spec = desired.Spec
```

Comment:
you diff meta, but you update only spec.

Reply:
Nice catch, thanks!  Done.